src/task_types/burn_in_tests.rs

```diff
@@ -461,14 +452,18 @@ impl BurnInService for BurnInServiceImpl {
             .expansions
             .insert(COMPILE_VARIANT.to_string(), compile_build_variant_name);
 
+        let large_distro = self
```

Comment:
[nit] Same comment as above, this could just be the regular distro right?

Reply:
Answered above.